.github/workflows/deploy.yaml

```diff
@@ -22,6 +22,37 @@ jobs:
           cd /home/ubuntu/
           ./deploy.sh
 
+  run-tests: 
+    needs: deploy-staging
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.10.8]
+    env:
+      MAIL_ID: ${{ secrets.GMAIL_USERNAME }}
+      PASSWORD: ${{ secrets.GMAIL_PASSWORD }}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          set -ex
+          wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+          sudo apt install ./google-chrome-stable_current_amd64.deb        
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install msedge-selenium-tools
+      - uses: nanasess/setup-chromedriver@v1
+      - run: chromedriver --url-base=/wd/hub &
+      - name: Run Tests
+        run: |
+          cd tests/integration
+          pip install -r requirements.txt
+          python -m pytest -n 3 -v --browser headless-chrome
```

Comment:
Check if verbose is printing something when you execute?

Reply:
No it is not printing anything. . .